src/Backend/Modules/MediaLibrary/Installer/Data/locale.xml

```diff
@@ -266,6 +266,22 @@
         <translation language="en"><![CDATA[back to overview]]></translation>
         <translation language="nl"><![CDATA[terug naar overzicht]]></translation>
       </item>
+      <item type="label" name="Crop">
```

Comment:
This translation belongs to < Backend >< Core >, otherwise the label isn't visible in other modules. Example of this in screenshot:
<img width="226" alt="schermafbeelding 2017-06-07 om 16 40 48" src="https://user-images.githubusercontent.com/588616/26884449-13833c1c-4ba0-11e7-8f32-b63f04f6da3d.png">